### DISCUSSION TITLE
Installing dotnet has failed!

### DISCUSSION THREAD

**Discussion by arbyruns** [Homebrew/discussions#1569] [Everyday usage]
I'm trying to run `brew bundle install dotnet` and running into the following error 

`Error: dotnet: no bottle available!
You can try to install from source with:
  brew install --build-from-source dotnet
Please note building from source is unsupported. You will encounter build
failures with some formulae. If you experience any issues please create pull
requests instead of asking for help on Homebrew's GitHub, Twitter or any other
official channels.  
Installing dotnet has failed!  
Using xclogparser
Using xcodegen
Using cocoapods
Using carthage
Using cocoapods
Homebrew Bundle failed! 1 Brewfile dependency failed to install.`

My brew version is: 

`
Homebrew 3.1.9 
Homebrew/  
Homebrew/homebrew-cask (git revision b4208e3d59; last commit 2021-05-27)  
`

Any thoughts?

**Comment by carlocab** [comment]
What's the output of `brew config` and `brew doctor`?

**Reply by arbyruns** [reply]
Brew config

OMEBREW_VERSION: 3.1.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: 76cfe88b7628a8010bf98115627f6aced4ff8870
Last commit: 4 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 011b8ed3b7a244f2333ef1eb4daa4a6a6411e7fc
Core tap last commit: 55 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 12.0.0 build 1200
Git: 2.31.1 => /opt/homebrew/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.3.1-arm64
CLT: 12.5.0.0.1.1617976050
Xcode: 12.4
Rosetta 2: false

and brew doctor is just telling me to update Xcode to 12.5, but we haven't moved to 12.5 yet.

**Reply by carlocab** [reply]
`dotnet` doesn't work on ARM yet. You can run it under Rosetta, but you'll need a Rosetta install of `brew` in `/usr/local`.

**Reply by arbyruns** [reply]
Thanks! Is this a good walk through on how to do this? https://stackoverflow.com/questions/64882584/how-to-run-the-homebrew-installer-under-rosetta-2-on-m1-macbook/64883440

**Reply by carlocab** [reply]
Yes, that's fine.

**Reply by arbyruns** [reply]
Hmmm that works, but when I switch back to my M1 version of homebrew it still does not find dotnet.

**Reply by carlocab** [reply]
That's by design. Your M1 version of `brew` knows nothing about your `/usr/local` version of `brew`.
